lib/DataFetcher.dart

```diff
@@ -73,8 +76,46 @@ class DataFetcher {
     return new List();
   }
 
-  static Future<bool> uploadGrades(String filePath) {
-    return Future.value(true); // TODO send to server
+  static fetchFormData(FormClass.FormData data) async {
+    Dio dio = new Dio();
+
+    final String serverEndPoint = _api + "/user/profile";
+
+    FormData formData = new FormData.fromMap(({
+      "name": "Christina Kreza", //Change age with name
```

Comment:
This needs to change. Why do you need to create a new FormData object since we already have one as a function argument?